common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -141,6 +141,32 @@ def test_add_entitlement(self):
         )
         assert results == CourseEntitlementSerializer(course_entitlement).data
 
+    def test_add_duplicate_entitlement(self):
+        """
+        Request with identical course_uuid and order_number should not create duplicate
+        entitlement
+        """
+        course_uuid = uuid.uuid4()
+        entitlement_data = self._get_data_set(self.user, str(course_uuid))
+
+        response = self.client.post(
+            self.entitlements_list_url,
+            data=json.dumps(entitlement_data),
+            content_type='application/json',
+        )
+        assert response.status_code == 201
```

Comment:
can we use assertEqual?

Reply:
we can, I was just following the convention used through out other tests in this module.

Reply:
I believe using the `==` and the assert is the preferred method now.